src/main/scala/org/apache/spark/sql/TiContext.scala

```diff
@@ -15,21 +15,35 @@
 
 package org.apache.spark.sql
 
+
+import com.pingcap.tikv.tools.RegionUtils
+
 import com.pingcap.tikv.{TiConfiguration, TiSession}
 import com.pingcap.tispark._
 import org.apache.spark.SparkConf
 import org.apache.spark.internal.Logging
+import scala.collection.JavaConverters._
 
 class TiContext(val session: SparkSession) extends Serializable with Logging {
   val sqlContext: SQLContext = session.sqlContext
   val conf: SparkConf = session.sparkContext.conf
   val tiConf: TiConfiguration = TiUtils.sparkConfToTiConf(conf)
   val tiSession: TiSession = TiSession.create(tiConf)
   val meta: MetaManager = new MetaManager(tiSession.getCatalog)
+
+  val debug: DebugTool = new DebugTool
+
   TiUtils.sessionInitialize(session, tiSession)
 
+
   final val version: String = TiSparkVersion.version
 
+  class DebugTool {
+    def getRegionDistribution(dbName: String, tableName: String): Map[String, Integer] = {
```

Comment:
Maybe adding some brief comment on `getRegionDistribution` here will be clearer for user to use.